guide/src/python_from_rust.md

```diff
@@ -236,6 +236,75 @@ def leaky_relu(x, slope=0.01):
 # }
 ```
 
+### Want to embed Python in Rust with exposed modules?
+
+Internally Python maintains the `sys.modules` dict.
+An import in Python is simply a lookup into this dict.
+
+It is necessary to insert the new module into the dict
+before it is exposed to Python. There are two ways of
+doing this.
+
+   * manually inserting into `sys.modules`
```

Comment:
```suggestion
   * creating the module using `PyModule::new` and manually inserting it into `sys.modules`
```

Reply:
Again this seems not to be needed (for me).

Reply:
See above.